src/main/scala/cromwell/binding/package.scala

```diff
@@ -21,8 +21,9 @@ package object binding {
   type WorkflowCoercedInputs = Map[FullyQualifiedName, WdlValue]
   type WorkflowOutputs = Map[FullyQualifiedName, WdlValue]
   type FullyQualifiedName = String
+  type LocallyQualifiedName = String
```

Comment:
Agreed in terms of removing old ones but why create more debt here? There's no harm in doing it the future-correct-way here.